### PR TITLE
feat: Add new buttons

### LINKED ISSUE #290: Feature Request(s): Additional Links | VRChat
### Please enter the brand name.
VRChat

### Please enter the brand website.
https://vrchat.com

### Please enter the brand SVG. (If Applicable)
https://github.com/sethcottle/littlelink/blob/main/images/icons/vrchat.svg

### LINKED ISSUE #407: [BRAND REQUEST] Pixelfed
### Please enter the brand name.

Pixelfed

### Please enter the brand website.

https://pixelfed.org/

### Please enter the brand SVG. (If Applicable)

https://w1.pixelcdn.net/assets/images/pixelfed-full-color.svg

### LINKED ISSUE #429: [BRAND REQUEST] Lemmy
### Please enter the brand name.

Lemmy

### Please enter the brand website.

https://join-lemmy.org

### Please enter the brand SVG. (If Applicable)

https://join-lemmy.org/static/assets/images/lemmy.svg

### PR DESCRIPTION
**Pull request by KieranRobson**
# Proposed Changes

- Added Lemmy - Closes #429 
- Added Pixelfed - Closes #407 
- Added VRCHat - Closes #290 
- Remove links for technotim.live due to this now leading to blog rather than link site.

Screenshot (If Applicable)
image

## Checklist

- [x] Tested locally
- [x] Ran `yarn ci` to test my code
- [x] Did not add any unnecessary changes
- [x] All my changes appear after other changes in each file
- [x] Included a screenshot (if adding a new button)
- [x] 🚀

### CODE REVIEW

**Comment by Babasile** [comment]
Good catch! Instead of removing you can replace https://technotim.live/ with https://links.technotim.live/

File: README.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ Inspired by [littlelink](https://github.com/sethcottle/littlelink).
...
 
-It takes the same simple approach to a link page and hosts it within a NodeJS server with React Server Side Rendering, containerized for you to use. Now, customizing `LittleLink` with `littlelink-server` is as easy as passing in some environment variables. If you need help configuring this, please see this [video](https://youtu.be/42SqfI_AjXU) at explains everything and a live example at [technotim.live](https://technotim.live/).
+It takes the same simple approach to a link page and hosts it within a NodeJS server with React Server Side Rendering, containerized for you to use. Now, customizing `LittleLink` with `littlelink-server` is as easy as passing in some environment variables. If you need help configuring this, please see this [video](https://youtu.be/42SqfI_AjXU) at explains everything.
```
Comment (timothystewart6) [inline README.md:14]: Thanks for noticing!  The new live example is at https://links.technotim.live

File: README.md [hunk 2]
```diff
@@ -25,8 +25,6 @@ It takes the same simple approach to a link page and hosts it within a NodeJS se
 
 Check the [docker-compose.yml](/docker-compose.yml) file for all supported buttons and configuration!
 
-The example below will generate a site exactly like <https://technotim.live>
```
Comment (timothystewart6) [inline README.md:28]: Thanks for noticing! The new live example is at https://links.technotim.live

**Review by timothystewart6 — CHANGES_REQUESTED** [review]
A few small tweaks for the links!  Thank you so much!

**Comment by KieranRobson** [comment]
All done!

**Review by timothystewart6 — APPROVED** [review]

**Comment by timothystewart6** [comment]
Thank you!

**Pull request merged**